### PR TITLE
Adds Powershell Logging Disable Procedures

### PR DESCRIPTION
**Pull request by w0rk3r**

### CODE REVIEW

**Comment by khyberspache** [comment]
Assigning myself, will review tomorrow AM.

File: defense-evasion/11c8c5a7-81d2-43af-8b87-42cba6259790.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+id: 11c8c5a7-81d2-43af-8b87-42cba6259790
+name: Disable Powershell Module Logging
+description: |
+  Module logging records pipeline execution details as PowerShell executes, including variable initialization and command invocations.
+  Attackers can disable logging to cover their activities on the host.
+metadata:
+  version: 1
+  authors:
+  - w0rk3r
+  tags: []
```
Comment (khyberspache) [inline defense-evasion/11c8c5a7-81d2-43af-8b87-42cba6259790.yml:10]: Add a `-destructive` to all of these TTPs under the `tags`

**Review by khyberspache — CHANGES_REQUESTED** [review]
I tried all three of these on a latest release of Win10Pro 21H1 and the reg key didn't exist

**Comment by w0rk3r** [comment]
All of them must be configured before in the Local Group Policy Editor, the keys don't exist if it is not configured

Before:
image


After:
image

File: defense-evasion/11c8c5a7-81d2-43af-8b87-42cba6259790.yml (hunk 1 above)
Comment (w0rk3r) [reply defense-evasion/11c8c5a7-81d2-43af-8b87-42cba6259790.yml]: Done

**Review by khyberspache — APPROVED** [review]

**Pull request merged**